src/resampling.jl

```diff
@@ -917,8 +918,12 @@ untouched.
 
 - `verbosity` level, an integer defaulting to 1.
 
-- `check_measure` - default is `true`
+- `check_measure` - default is `true`.
 
+- `logger` - Binds a logging platform. It receives an instance object with
+  the required client parameters. Default is `nothing`.
+
+See [Loggers](@ref) for logger details.
```

Comment:
```suggestion
  the required client parameters. Default is `nothing`. See [Loggers](@ref) for logger details.
```

Also, I tried `?Loggers` and didn't get anything.  And `MLJBase.Loggers` does not exist.  I guess this is coming?